crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -484,15 +916,46 @@ impl GetBatchData for MeshPipeline {
     }
 }
 
-impl GetBinnedBatchData for MeshPipeline {
-    type Param = (SRes<RenderMeshInstances>, SRes<RenderLightmaps>);
+impl GetFullBatchData for MeshPipeline {
+    type BufferInputData = MeshInputUniform;
 
-    type BufferData = MeshUniform;
+    fn get_batch_input_index(
```

Comment:
Maybe `get_batch_input_and_index`?

Reply:
It's the index *of* the `MeshInputUniform`. But I can see how the name could be confusing. Suggestions are welcome. Maybe `get_index_of_batch_input`?

Reply:
I haven't heard from you, so I'm just going to go ahead and name it `get_index_of_batch_input`.